packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -183,6 +183,15 @@ export const buildKonnectorsQueryByQualificationLabel = label => ({
   }
 })
 
+export const buildKonnectorsQueryBySlug = slug => ({
+  definition: () => Q(KONNECTORS_DOCTYPE).where({ slug }).indexFields(['slug']),
```

Comment:
You should use getById(`io.cozy.konnectors/${slug}` instead of the `where` . 

No need of index, no need of `as` attribute. Lot of gains ;)

Reply:
(ccl: we can't use getById ATM since it throws an error when the app is not installed)